packages/components/Shadow/src/Shadow.types.ts

```diff
@@ -0,0 +1,22 @@
+import { ViewProps } from 'react-native';
+import { IBorderTokens, IColorTokens, LayoutTokens } from '@fluentui-react-native/tokens';
+
+export const shadow = 'Shadow';
+
+export type ShadowDepth = '2' | '4' | '8' | '16' | '28' | '64';
+
+export interface ShadowTokens extends LayoutTokens, IBorderTokens, IColorTokens {}
+
+export interface ShadowProps extends ViewProps, IBorderTokens {
+  depth?: ShadowDepth;
+}
```

Comment:
I would delete everything that is unused in this file. Also, make it not extend IBorderTokens